src/pages/GrossProfit/GrossProfit.tsx

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2007-2019, GoodData(R) Corporation. All rights reserved.
+import '@gooddata/react-components/styles/css/main.css';
+import React, { FunctionComponent } from 'react';
+import GrossProfitAllMonths from '../../components/GrossProfit/GrossProfitAllMonths';
+import GrossProfitMonth from '../../components/GrossProfit/GrossProfitMonth';
+import { PROJECT_ID } from '../../constants';
+import { MEASURES, VIEW_BY } from '../../utils/helpers';
+
+const GrossProfit: FunctionComponent = () => (
+  <div>
+    <GrossProfitMonth measures={MEASURES} projectId={PROJECT_ID} />
```

Comment:
`measures` and `projectId` should be passed from parent component so that `GrossProfit` can be used to render different contents.